pkg/doc/signature/signer/signer.go

```diff
@@ -103,7 +103,7 @@ func (signer *DocumentSigner) signObject(context *Context, jsonLdObject map[stri
 		Type:                    context.SignatureType,
 		SignatureRepresentation: context.SignatureRepresentation,
 		Creator:                 context.Creator,
-		Created:                 created,
+		Created:                 &util.TimeWithTrailingZeroMsec{Time: *created},
```

Comment:
Would be helpful to have a New() constructor rather than creating the struct.

Reply:
Good idea, I will take a look.

Reply:
Added `NewTime()` and `NewTimeWithTrailingZeroMsec()` factory methods.

Reply:
... but still pending to use it here @kdimak